CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file based on the
 
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.0...HEAD)
 
+### Backward Compatibility Breaks
+- All classes that depend on \Elastica\Client by typehint have been updated to accept a ClientInterface. Any classes that extend Elastica classes that depend on this typehint will need to be updated.
```

Comment:
Being the entry point to Elastica, enough I think - 

https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Elastica/Client.php